packages/sdk-v2/src/sdk/goodid/types.ts

```diff
@@ -0,0 +1,30 @@
+// Reference: https://veramo.io/docs/basics/verifiable_data
+export interface CredentialProof {
+  type: string;
+  jwt: string;
+}
+
+export interface CredentialSubject {
+  id: string;
+  [key: string]: any;
+}
+
+export interface Issuer {
+  id: string;
+}
+
+export interface VerifiableCredential {
+  credentialSubject: CredentialSubject;
+  issuer: Issuer;
+  type: string[];
+  "@context": string[];
+  issuanceDate: string;
+  proof: CredentialProof;
+}
+
+// Might change
+export enum CredentialTypes {
```

Comment:
there's also VerifiableIdentityCredential, I've linked the list above

Reply:
added